samples/bluetooth/mesh/light/README.rst

```diff
@@ -132,7 +132,10 @@ This sample is split into the following source files:
 DFU configuration
 =================
 
```

Comment:
Maybe we can consider using tabs in this section for the different DKs instead of subsections. Either one tab pr supported DK, or just two tabs describing the two different ways of doing it, for nRF52/nRF54L Series and nRF5340.
E.g. check this section: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/installation/install_ncs.html#set-up-the-command-line-build-environment

What do you think?